src/Transport/Curl.php

```diff
@@ -383,7 +382,7 @@ private function setup_handle($url, $headers, $data, $options) {
 		$options['hooks']->dispatch('curl.before_request', [&$this->handle]);
 
 		// Force closing the connection for old versions of cURL (<7.22).
```

Comment:
This comment is no longer valid.

Reply:
The comment was already present before we made the problematic change and was the reason why we used the specific version constraint we ended up with. Before we fully understand the context and where this was originally coming for, I'll prefer to keep in in there.

Reply:
Yes, indeed, and I think that's what has already happened in this case.
But I'm still assuming there is a reason why this was put in there initially. I'd like to understand this better before removing it.

Reply:
For future reference, this is the PR that introduced the comment: https://github.com/WordPress/Requests/pull/211